.github/workflows/build_dev.yml

```diff
@@ -76,37 +79,55 @@ jobs:
           kubectl patch mpo ${MODULES_MODULE_NAME} --type merge -p '{"spec":{"imageTag":"${MODULES_MODULE_TAG}"}}'
 
           OUTER
-        shell: bash
 
-  lint:
+  lint_go:
     runs-on: ubuntu-latest
-    name: Run linter
+    name: Run go linter
     steps:
-      - name: Set up Go 1.20
+      - name: Set up Go 1.21
         uses: actions/setup-go@v5
         with:
-          go-version: "1.20"
+          go-version: "1.21"
       
       - name: Install Task
         uses: arduino/setup-task@v1
-      
+        
       - uses: actions/checkout@v4
-      
+        
       - name: Run lint virtualization-controller
         run: |
           task virtualization-controller:init
           task virtualization-controller:lint:go
-        shell: bash
-        continue-on-error: true
+  
+  lint_yaml:
+    runs-on: ubuntu-latest
+    name: Run yaml linter
+    steps:
+      - name: Set up Go 1.21
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.21"
+      
```

Comment:
```suggestion
```

Go is not needed for prettier.

Reply:
Go needed for run task, prettier running by task